domain/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Remove this file

Reply:
Weird, I don't see this file locally. Let's see if it disappears upon my next push.